code/modules/paperwork/sticky_note.dm

```diff
@@ -0,0 +1,92 @@
+/obj/item/paper_bin/pad
+	name = "sticky note pad"
+	desc = "A pad of densely packed sticky notes."
+	icon = 'icons/obj/stickynotes.dmi'
+	icon_state = "pad"
+	papertype = /obj/item/paper/sticky
+	w_class = WEIGHT_CLASS_SMALL
+	holds_pen = FALSE
+
+/obj/item/paper_bin/pad/examine(mob/user)
+	. = ..()
+	. += span_notice("Write notes on top using a <b>pen</b>.")
+
+
+/obj/item/paper_bin/pad/Initialize()
+	papertype = pick(typesof(/obj/item/paper/sticky))
+	. = ..()
+
+/obj/item/paper_bin/pad/attackby(obj/item/thing, mob/user)
+	. = ..()
+	if(istype(thing, /obj/item/pen))
+		var/obj/item/paper/top_paper = papers[papers.len]
```

Comment:
Can we not pass the attackby to the paper instead of replicating writecode here?

Reply:
I prefer how fast this is compared to taking the paper out and writing on it.